custom/icds_reports/ucr/expressions.py

```diff
@@ -346,6 +347,65 @@ def __str__(self):
         return "User's location id"
 
 
+class AWCOwnerId(JsonObject):
+    type = TypeProperty('icds_awc_owner_id')
+    case_id_expression = DefaultProperty(required=True)
+
+    def configure(self, case_id_expression):
+        self._case_id_expression = case_id_expression
+
+    def __call__(self, item, context=None):
+        case_id = self._case_id_expression(item, context)
+
+        if not case_id:
+            return None
+
+        if item['owner_id'] and item['owner_id'] != '-':
```

Comment:
Is `-` already an established convention? Is it different than the `---` used elsewhere?

Reply:
Tempted to suggest putting this check inside something more readable, like an `ownerIdExists` helper.

Reply:
I believe that `---` is purely a UI/export thing that means a property doesn't exist? 

`-` is the default owner for extension cases and will be followed as a convention in REACH (when save to case is necessary), but that could be pulled out into a constant